### ISSUE TITLE
Xwrapper not working?

### ISSUE THREAD

**Issue by cyberkitsune** [neutrinolabs/xrdp#1512]
Hey,

I'm on manjaro with a fresh install of the latest xrdp and xorgxrdp. 
I made the file `/etc/X11/Xwrapper.config` and it contains:
```
allowed_users=anybody
```

I can connect to xrdp from windows and see the login screen, but when I try to log in, it just hangs there. The log files show that the Xserver it spawns instantly closes.

The xorg log shows:
`parse_vt_settings: Cannot open /dev/tty0 (Permission denied)`
As the reason for it not working.

Any insight here? Other solutions seem to point to the xwrapper, but it seems that it's not working?

**Comment by matt335672** [comment]
Not really my area, but have a look at #1159 

In particular, check xorgxrdp is installled correctly.

**Comment by cyberkitsune** [comment]
> Not really my area, but have a look at #1159
> 
> In particular, check xorgxrdp is installled correctly.

Was just checking that thread. I've confirmed `/etc/X11/Xwrapper.config` has been edited, and I double checked that xorgxrdp is installed. (The config files at `/etc/X11/xrdp/xorg.conf` exist, as an example.)

**Comment by jsorg71** [comment]
an easy way to check Xorg is starting ok, run
Xorg -config xrdp/xorg.conf -noreset :10

**Comment by cyberkitsune** [comment]
> Xorg -config xrdp/xorg.conf -noreset :10

This gives me
```
Current version of pixman: 0.38.4
        Before reporting problems, check http://wiki.x.org
        to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
        (++) from command line, (!!) notice, (II) informational,
        (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(==) Log file: "/var/log/Xorg.10.log", Time: Wed Mar 11 13:24:37 2020
(++) Using config file: "/etc/X11/xrdp/xorg.conf"
(==) Using config directory: "/etc/X11/xorg.conf.d"
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
xorgxrdpSetup:
xrdpdevSetup:
rdpIdentify:
rdpDriverFunc: op 10
rdpProbe:
(EE) 
Fatal server error:
(EE) no screens found(EE) 
(EE) 
Please consult the The X.Org Foundation support 
         at http://wiki.x.org
 for help. 
(EE) Please also check the log file at "/var/log/Xorg.10.log" for additional information.
(EE) 
(EE) Server terminated with error (1). Closing log file.
```

Huh. No screens found? Separate issue or related?

**Comment by metalefty** [comment]
Did you see `sesman.ini`?

```
;
; Session definitions - startup command-line parameters for each session type
;

[Xorg]
; Specify the path of non-suid Xorg executable. It might differ depending
; on your distribution and version. The typical path is shown as follows:
;
; Fedora 26 or later    :  param=/usr/libexec/Xorg
; Debian 9 or later     :  param=/usr/lib/xorg/Xorg
; Ubuntu 16.04 or later :  param=/usr/lib/xorg/Xorg
; Arch Linux            :  param=/usr/lib/xorg-server/Xorg
; CentOS 7              :  param=/usr/bin/Xorg or param=Xorg
;
param=Xorg
; Leave the rest paramaters as-is unless you understand what will happen.
```

**Comment by metalefty** [comment]
What I intended to say is: you should bypass Xwrapper rather than  doing trial & error on Xrapper. `sesman.ini` have instructions how to do that.

**Comment by jsorg71** [comment]
Later Xorg has another option in /etc/X11/Xwrapper.config
needs_root_rights=no
Ideally, you don't want your Xorg(for xrdp) running as root
I'm not sure what version Manjaro uses.

**Comment by metalefty** [comment]
Regarding Manjaro, fixed by #1529 .
